src/Symfony/Component/Form/Tests/ButtonBuilderTest.php

```diff
@@ -47,9 +47,6 @@ public function testNameContainingIllegalCharacters()
         $this->assertInstanceOf('\Symfony\Component\Form\ButtonBuilder', new ButtonBuilder('button[]'));
     }
 
-    /**
-     * @group legacy
```

Comment:
This one should logically be kept, see https://github.com/symfony/symfony/pull/34445.

Reply:
fixed in #34448

Reply:
@nicolas-grekas can you rebase now that the other PR removed this test entirely ?

Reply:
rebased